src/backend/dx12/src/lib.rs

```diff
@@ -919,6 +919,9 @@ impl hal::Instance for Instance {
                     min_texel_buffer_offset_alignment: 1, // TODO
                     min_uniform_buffer_offset_alignment: 256, // Required alignment for CBVs
                     min_storage_buffer_offset_alignment: 1, // TODO
+                    framebuffer_color_samples_count: 0b101,
```

Comment:
any reason this only supports 4xMSAA?

Reply:
It's the minimum supported sample count for all render target formats. Every format must support at least these sample counts ( 1 and 4 ).

Reply:
I see. Let's put a `//TODO` to expose more sample counts in the future?